src/components/DialogExportStatement.js

```diff
@@ -132,13 +134,45 @@ const DialogExportStatement = ({ dialogOpen, onCloseHandler }) => {
     if (
       startDate > endDate ||
       startDate.invalid !== null ||
-      endDate.invalid !== null
+      endDate.invalid !== null ||
+      errorFromInput !== null ||
+      errorToInput !== null
     ) {
       setIsDisabled(true);
     } else {
       setIsDisabled(false);
     }
-  }, [startDate, endDate]);
+  }, [startDate, endDate, errorFromInput, errorToInput]);
+
+  const errorMessageTo = useMemo(() => {
+    switch (errorToInput) {
+      case 'maxDate':
+      case 'minDate': {
+        return 'Select a correct date range';
+      }
+      case 'invalidDate': {
+        return 'Your date is not valid';
```

Comment:
See comments of new messages. We can keep the logic for now. But the two messages needs to be in en.json, not defined here!